Experiments_toolkit.py

```diff
@@ -76,35 +76,32 @@ def preprocess_data(DATA, rows, filtering_params, window_size, offset):
                                 these windows (numpy arrays) into a dictionary.
 
        Args:
-           DATA -- dict (Dictionary of keys and within the keys are the 3D data samples)
-           rows -- int (Number of rows in the data)
-           filtering_params -- All_Filtering_Params (Dataclass found in windowing.py that 
+        	DATA -- np array (Numpy array where the last row is time steps)
```

Comment:
for an np array, it's good to specify what dimensions are expected, e.g. [(sensor_dims+1), num_samples]